src/js/common/dom.js

```diff
@@ -316,7 +323,16 @@ class DOM {
       }
 
       if (value) {
-        element.setAttribute(name, value)
+        // if multiple options are being used and they are being selected automatically, the browser wants the element
```

Comment:
@kevinchappell 
This is because the browser will only do something with this attribute when the form has been actually rendered. Using setTimeout makes sure the attribute is set on the next iteration of the event loop, when the form has been rendered